.github/workflows/pull-request.yml

```
@@ -204,6 +203,73 @@ jobs:
+            --exclude 'dart:async,dart:collection,dart:convert,dart:core,dart:developer,dart:io' \
+
+      - name: Convert HTML to Markdown
+        run: |
+          python .github/workflows/convert_html_to_md.py
+
+      - name: Clean up temporary files
+        run: |
+          rm -rf docs/temp-docs
+
+      - name: Commit and push changes
+        run: |
```

actionlint (1.7.4)

"github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)